### PR TITLE
Use Python 3.10 everywhere

### LINKED ISSUE #607: Update references to Python 3.9
## Description

In #502 we updated Python to 3.10, but there are still a few references to 3.9:

```
$ rg "3\.9"
openverse_catalog/templates/README.md
7:To add a Provider API script using this template, you will need to have Python 3 installed on your machine (preferably, version 3.9). You will also need to know the name of provider, and the type of media you are going to collect (`image` or `audio`).

docker/airflow/Dockerfile
49:ARG CONSTRAINTS_FILE="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-3.9.txt"

requirements.txt
3:flake8==3.9.2

tests/dags/providers/provider_api_scripts/resources/clevelandmuseum/handle_response_data.json
2951:      "updated_at": "2020-01-13 09:00:23.979000",
```

_Note the Dockerfile and README.md_

The most important of those two is the constraints file, as it dictates which libraries are compatible with both Airflow and Python for specific versions.

## Additional context

Note: This will require a deployment since the DAG sync script only updates DAG files and this will require a change to the Docker image.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### LINKED ISSUE #655: Update or remove the provider script templates
## Current Situation

There are provider templates and instructions for using them that do not use the new provider script class.

## Suggested Improvement

Either update the templates or remove them entirely in favor of just using the class.

## Benefit

Consistency and clarify in the intention of approach to new provider scripts.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

Fixes #607 by @AetherUnbound 
Fixes #655 by @sarayourfriend 

## Description

Centralises the Python version declaration in a single place so that it can't drift again. Also sets it to 3.10 as the issue required.

I also went ahead and removed the template files as I don't think they're being used anymore. I can undo this bit if necessary.

## Testing Instructions

Checkout the branch and run `just recreate` and confirm that it uses 3.10 for everything, the base image and the constraints file. Confirm that everything installs correctly and runs well (CI will confirm this as well).

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by obulat — CHANGES_REQUESTED** [review]
Oops, I just noticed that the tests for the templates also need to be deleted to pass the CI.

**Review by obulat — APPROVED** [review]
LGTM!

**Review by dhruvkb — APPROVED** [review]
👏 This PR has some very innovative approaches to reuse the value defined in one file in so many different places.

We should do the same for `WordPress/openverse-api` as well.

**Comment by sarayourfriend** [comment]
Waiting on @AetherUnbound or @stacimc to review this and confirm the approach before merging.

File: justfile
Comment (sarayourfriend) [inline justfile:19]:
```suggestion
# Check the installed Python version matches the required Python version and fail if not
```

**Review by AetherUnbound — APPROVED** [review]
I love this approach! Once I rebased this on main and fixed the conflicts, all of the tests passed. I'm happy to push that rebase if you'd like.

I am getting an error when I try to run `just run python -v` 🤔 never seen anything like it, I'll try to look into it a bit more. The way it's behaving for me, it would seem to be a permissions issue that'd persist for v3.9 too, or any recent iteration of our docker image.

<details><summary>Output of `just run python -v`</summary>

```
####################################################################################################################################################################################################################
                                                                                                RUNNING "python -v"
####################################################################################################################################################################################################################
import _frozen_importlib # frozen
import _imp # builtin
import '_thread' # <class '_frozen_importlib.BuiltinImporter'>
import '_warnings' # <class '_frozen_importlib.BuiltinImporter'>
import '_weakref' # <class '_frozen_importlib.BuiltinImporter'>
import '_io' # <class '_frozen_importlib.BuiltinImporter'>
import 'marshal' # <class '_frozen_importlib.BuiltinImporter'>
import 'posix' # <class '_frozen_importlib.BuiltinImporter'>
import '_frozen_importlib_external' # <class '_frozen_importlib.FrozenImporter'>
# installing zipimport hook
import 'time' # <class '_frozen_importlib.BuiltinImporter'>
import 'zipimport' # <class '_frozen_importlib.FrozenImporter'>
# installed zipimport hook
# code object from /usr/local/lib/python3.10/encodings/__init__.py
# could not create '/usr/local/lib/python3.10/encodings/__pycache__/__init__.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/codecs.py
# could not create '/usr/local/lib/python3.10/__pycache__/codecs.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_codecs' # <class '_frozen_importlib.BuiltinImporter'>
import 'codecs' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d002d150>
# code object from /usr/local/lib/python3.10/encodings/aliases.py
# could not create '/usr/local/lib/python3.10/encodings/__pycache__/aliases.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'encodings.aliases' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0067c40>
import 'encodings' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d01bf6a0>
# code object from /usr/local/lib/python3.10/encodings/utf_8.py
# could not create '/usr/local/lib/python3.10/encodings/__pycache__/utf_8.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'encodings.utf_8' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d006c190>
import '_signal' # <class '_frozen_importlib.BuiltinImporter'>
# code object from /usr/local/lib/python3.10/io.py
# could not create '/usr/local/lib/python3.10/__pycache__/io.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/abc.py
# could not create '/usr/local/lib/python3.10/__pycache__/abc.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_abc' # <class '_frozen_importlib.BuiltinImporter'>
import 'abc' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d006e0b0>
import 'io' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d006c4c0>
# code object from /usr/local/lib/python3.10/site.py
# could not create '/usr/local/lib/python3.10/__pycache__/site.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/os.py
# could not create '/usr/local/lib/python3.10/__pycache__/os.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/stat.py
# could not create '/usr/local/lib/python3.10/__pycache__/stat.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_stat' # <class '_frozen_importlib.BuiltinImporter'>
import 'stat' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0070220>
# code object from /usr/local/lib/python3.10/_collections_abc.py
# could not create '/usr/local/lib/python3.10/__pycache__/_collections_abc.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_collections_abc' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0035f30>
# code object from /usr/local/lib/python3.10/posixpath.py
# could not create '/usr/local/lib/python3.10/__pycache__/posixpath.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/genericpath.py
# could not create '/usr/local/lib/python3.10/__pycache__/genericpath.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'genericpath' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d007a290>
import 'posixpath' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00c4700>
import 'os' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00962f0>
# code object from /usr/local/lib/python3.10/_sitebuiltins.py
# could not create '/usr/local/lib/python3.10/__pycache__/_sitebuiltins.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_sitebuiltins' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00d9540>
Processing user site-packages
Adding directory: '/usr/local/airflow/.local/lib/python3.10/site-packages'
Processing global site-packages
Adding directory: '/usr/local/lib/python3.10/site-packages'
Processing .pth file: '/usr/local/lib/python3.10/site-packages/distutils-precedence.pth'
import 'site' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0035b10>
Python 3.10.1 (main, Dec 21 2021, 09:17:12) [GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
# extension module 'readline' loaded from '/usr/local/lib/python3.10/lib-dynload/readline.cpython-310-x86_64-linux-gnu.so'
# extension module 'readline' executed from '/usr/local/lib/python3.10/lib-dynload/readline.cpython-310-x86_64-linux-gnu.so'
import 'readline' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7fb7d007b6d0>
import 'atexit' # <class '_frozen_importlib.BuiltinImporter'>
# code object from /usr/local/lib/python3.10/rlcompleter.py
# could not create '/usr/local/lib/python3.10/__pycache__/rlcompleter.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/inspect.py
# could not create '/usr/local/lib/python3.10/__pycache__/inspect.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/ast.py
# could not create '/usr/local/lib/python3.10/__pycache__/ast.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_ast' # <class '_frozen_importlib.BuiltinImporter'>
# code object from /usr/local/lib/python3.10/contextlib.py
# could not create '/usr/local/lib/python3.10/__pycache__/contextlib.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/collections/__init__.py
# could not create '/usr/local/lib/python3.10/collections/__pycache__/__init__.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'itertools' # <class '_frozen_importlib.BuiltinImporter'>
# code object from /usr/local/lib/python3.10/keyword.py
# could not create '/usr/local/lib/python3.10/__pycache__/keyword.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'keyword' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cff28250>
# code object from /usr/local/lib/python3.10/operator.py
# could not create '/usr/local/lib/python3.10/__pycache__/operator.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_operator' # <class '_frozen_importlib.BuiltinImporter'>
import 'operator' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cff285b0>
# code object from /usr/local/lib/python3.10/reprlib.py
# could not create '/usr/local/lib/python3.10/__pycache__/reprlib.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'reprlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cfec9f30>
import '_collections' # <class '_frozen_importlib.BuiltinImporter'>
import 'collections' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0017c40>
# code object from /usr/local/lib/python3.10/functools.py
# could not create '/usr/local/lib/python3.10/__pycache__/functools.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/types.py
# could not create '/usr/local/lib/python3.10/__pycache__/types.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'types' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d006cac0>
import '_functools' # <class '_frozen_importlib.BuiltinImporter'>
import 'functools' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00c6770>
import 'contextlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cfec98d0>
# code object from /usr/local/lib/python3.10/enum.py
# could not create '/usr/local/lib/python3.10/__pycache__/enum.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'enum' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d002e800>
import 'ast' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0065e10>
# code object from /usr/local/lib/python3.10/dis.py
# could not create '/usr/local/lib/python3.10/__pycache__/dis.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/opcode.py
# could not create '/usr/local/lib/python3.10/__pycache__/opcode.cpython-310.pyc': PermissionError(13, 'Permission denied')
# extension module '_opcode' loaded from '/usr/local/lib/python3.10/lib-dynload/_opcode.cpython-310-x86_64-linux-gnu.so'
# extension module '_opcode' executed from '/usr/local/lib/python3.10/lib-dynload/_opcode.cpython-310-x86_64-linux-gnu.so'
import '_opcode' # <_frozen_importlib_external.ExtensionFileLoader object at 0x7fb7cff98d00>
import 'opcode' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0071ff0>
import 'dis' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cfecb070>
# code object from /usr/local/lib/python3.10/collections/abc.py
# could not create '/usr/local/lib/python3.10/collections/__pycache__/abc.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'collections.abc' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cff99ae0>
# code object from /usr/local/lib/python3.10/importlib/__init__.py
# could not create '/usr/local/lib/python3.10/importlib/__pycache__/__init__.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/warnings.py
# could not create '/usr/local/lib/python3.10/__pycache__/warnings.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'warnings' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cff37fa0>
import 'importlib' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cff99840>
# code object from /usr/local/lib/python3.10/importlib/machinery.py
# could not create '/usr/local/lib/python3.10/importlib/__pycache__/machinery.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'importlib.machinery' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00d9f90>
# code object from /usr/local/lib/python3.10/linecache.py
# could not create '/usr/local/lib/python3.10/__pycache__/linecache.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/tokenize.py
# could not create '/usr/local/lib/python3.10/__pycache__/tokenize.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/re.py
# could not create '/usr/local/lib/python3.10/__pycache__/re.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/sre_compile.py
# could not create '/usr/local/lib/python3.10/__pycache__/sre_compile.cpython-310.pyc': PermissionError(13, 'Permission denied')
import '_sre' # <class '_frozen_importlib.BuiltinImporter'>
# code object from /usr/local/lib/python3.10/sre_parse.py
# could not create '/usr/local/lib/python3.10/__pycache__/sre_parse.cpython-310.pyc': PermissionError(13, 'Permission denied')
# code object from /usr/local/lib/python3.10/sre_constants.py
# could not create '/usr/local/lib/python3.10/__pycache__/sre_constants.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'sre_constants' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00a8a90>
import 'sre_parse' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cfecb370>
import 'sre_compile' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cff29300>
import '_locale' # <class '_frozen_importlib.BuiltinImporter'>
# code object from /usr/local/lib/python3.10/copyreg.py
# could not create '/usr/local/lib/python3.10/__pycache__/copyreg.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'copyreg' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cfd17f40>
import 're' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d0065600>
# code object from /usr/local/lib/python3.10/token.py
# could not create '/usr/local/lib/python3.10/__pycache__/token.cpython-310.pyc': PermissionError(13, 'Permission denied')
import 'token' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cff94d90>
import 'tokenize' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7cffa7d30>
import 'linecache' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00da710>
import 'inspect' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d00dbdf0>
import 'rlcompleter' # <_frozen_importlib_external.SourceFileLoader object at 0x7fb7d007b580>
>>> 
# clear builtins._
# clear sys.path
# clear sys.argv
# clear sys.ps1
# clear sys.ps2
# clear sys.last_type
# clear sys.last_value
# clear sys.last_traceback
# clear sys.path_hooks
# clear sys.path_importer_cache
# clear sys.meta_path
# clear sys.__interactivehook__
# restore sys.stdin
# restore sys.stdout
# restore sys.stderr
# cleanup[2] removing sys
# cleanup[2] removing builtins
# cleanup[2] removing _frozen_importlib
# cleanup[2] removing _imp
# cleanup[2] removing _thread
# cleanup[2] removing _warnings
# cleanup[2] removing _weakref
# cleanup[2] removing _io
# cleanup[2] removing marshal
# cleanup[2] removing posix
# cleanup[2] removing _frozen_importlib_external
# cleanup[2] removing time
# cleanup[2] removing zipimport
# destroy zipimport
# cleanup[2] removing _codecs
# cleanup[2] removing codecs
# cleanup[2] removing encodings.aliases
# cleanup[2] removing encodings
# destroy encodings
# cleanup[2] removing encodings.utf_8
# cleanup[2] removing _signal
# cleanup[2] removing _abc
# cleanup[2] removing abc
# cleanup[2] removing io
# cleanup[2] removing __main__
# cleanup[2] removing _stat
# cleanup[2] removing stat
# cleanup[2] removing _collections_abc
# cleanup[2] removing genericpath
# cleanup[2] removing posixpath
# cleanup[2] removing os.path
# cleanup[2] removing os
# cleanup[2] removing _sitebuiltins
# cleanup[2] removing site
# destroy site
# cleanup[2] removing readline
# cleanup[2] removing atexit
# cleanup[2] removing _ast
# destroy _ast
# cleanup[2] removing itertools
# cleanup[2] removing keyword
# destroy keyword
# cleanup[2] removing _operator
# cleanup[2] removing operator
# destroy operator
# cleanup[2] removing reprlib
# destroy reprlib
# cleanup[2] removing _collections
# cleanup[2] removing collections
# cleanup[2] removing types
# cleanup[2] removing _functools
# cleanup[2] removing functools
# cleanup[2] removing contextlib
# destroy contextlib
# cleanup[2] removing enum
# cleanup[2] removing ast
# cleanup[2] removing _opcode
# cleanup[2] removing opcode
# destroy opcode
# cleanup[2] removing dis
# cleanup[2] removing collections.abc
# cleanup[2] removing importlib._bootstrap
# cleanup[2] removing importlib._bootstrap_external
# cleanup[2] removing warnings
# cleanup[2] removing importlib
# cleanup[2] removing importlib.machinery
# cleanup[2] removing _sre
# cleanup[2] removing sre_constants
# destroy sre_constants
# cleanup[2] removing sre_parse
# cleanup[2] removing sre_compile
# cleanup[2] removing _locale
# cleanup[2] removing copyreg
# cleanup[2] removing re
# cleanup[2] removing token
# cleanup[2] removing tokenize
# cleanup[2] removing linecache
# cleanup[2] removing inspect
# cleanup[2] removing rlcompleter
# destroy rlcompleter
# destroy _signal
# destroy _sitebuiltins
# destroy posixpath
# destroy _stat
# destroy genericpath
# destroy stat
# destroy atexit
# destroy inspect
# destroy __main__
# destroy ast
# destroy dis
# destroy importlib
# destroy linecache
# destroy token
# destroy types
# destroy io
# destroy _opcode
# destroy warnings
# destroy importlib.machinery
# destroy os
# destroy tokenize
# destroy collections
# destroy itertools
# destroy re
# destroy enum
# destroy sre_compile
# destroy functools
# destroy copyreg
# destroy sre_parse
# destroy _sre
# destroy _locale
# cleanup[3] wiping importlib._bootstrap_external
# cleanup[3] wiping importlib._bootstrap
# cleanup[3] wiping collections.abc
# cleanup[3] wiping _functools
# cleanup[3] wiping _collections
# cleanup[3] wiping _operator
# cleanup[3] wiping readline
# cleanup[3] wiping _collections_abc
# cleanup[3] wiping abc
# cleanup[3] wiping _abc
# cleanup[3] wiping encodings.utf_8
# cleanup[3] wiping encodings.aliases
# cleanup[3] wiping codecs
# cleanup[3] wiping _codecs
# cleanup[3] wiping time
# cleanup[3] wiping _frozen_importlib_external
# cleanup[3] wiping posix
# cleanup[3] wiping marshal
# cleanup[3] wiping _io
# cleanup[3] wiping _weakref
# cleanup[3] wiping _warnings
# cleanup[3] wiping _thread
# cleanup[3] wiping _imp
# cleanup[3] wiping _frozen_importlib
# cleanup[3] wiping sys
# cleanup[3] wiping builtins
# destroy readline
# destroy posix
# destroy _frozen_importlib_external
# destroy _imp
# destroy io
# destroy marshal
# destroy time
# destroy _warnings
# destroy _frozen_importlib
# destroy codecs
# destroy encodings.aliases
# destroy encodings.utf_8
# destroy _codecs
# destroy abc
# destroy _abc
# destroy _collections_abc
# destroy sys
# destroy _weakref
# destroy _collections
# destroy collections.abc
# destroy builtins
# destroy _functools
# destroy _operator
# destroy _thread
# clear sys.audit hooks
```

</details>

**Pull request merged**
